third_party/blink/renderer/core/css/style_engine.cc

```diff
@@ -802,6 +802,11 @@ CSSStyleSheet* StyleEngine::CreateSheet(
 
   auto result = text_to_sheet_cache_.insert(text_content, nullptr);
   StyleSheetContents* contents = result.stored_value->value;
+
+  // #RUN-1065
```

Comment:
This comment doesn't really add anything.  I think given the inconsistency with which these comments are appearing now we should just have the bare assertion and make sure to include the issue reference in the assert.

Reply:
Fixed it up with a better comment about the problem.